solutions_builder/template_root/copier.yaml

```diff
@@ -23,6 +23,16 @@ gcp_region:
   help: Which Google Cloud region?
   default: us-central1
 
+gcp_zone:
```

Comment:
Refactor here:
- Change this variable to be jump_host_zone
- Move this after use_jump_host question
- Add a "when" so it only asks this question when use_jump_host is true.